examples/run_FS.py

```diff
@@ -1,5 +1,5 @@
-#{{{ note on phase cycling
-'''
+# {{{ note on phase cycling
```

Comment:
This is the field sweep at no power vs run_field_dep_mw.py is run at a set power